lib/screens/app/ecosystem/overview.dart

```diff
@@ -158,6 +166,10 @@ class _OverviewState extends State<Overview> {
 
   @override
   Widget build(BuildContext context) {
+
+    bool longText = true;
```

Comment:
long text and small text are very bad variable names. 
Use something descriptive. 

And why are these needed?

Reply:
Is this title and subtitle?

Reply:
Agreed...